Documentation/Upgrade/InstallTheNewSource/Index.rst

```diff
@@ -21,9 +21,9 @@ Upgrading a Composer Based System
 
 .. caution::
 
-   TYPO3 v9 LTS has no support for the package `typo3/cms` anymore - instead
+   TYPO3 v9 LTS has no support for the package `typo3/cms` anymore—instead
    all TYPO3 system extensions are now single packages. When upgrading, remove
-   the `typo3/cms` package and require the single packages you need.
+   the `typo3/cms` package and use require to fetch the single packages you need.
```

Comment:
```suggestion
   the `typo3/cms` package and procure the packages you need by fetching them individually.
```

Reply:
Here, would we put require in backticks?

Reply:
Is it a program?